inc/hcpp-async.h

```diff
@@ -163,6 +164,13 @@ inline void _asyncAwait(hclib_ddf_t ** ddf_list, T lambda) {
 }
 
 template <typename T>
+inline void asyncAwaitAt(hclib_ddf_t **ddf_list, place_t *pl, T lambda) {
```

Comment:
How about having place_t as first parameter and ddf arguments following this. In case of C++ interface, the user will pass "N" number of DDFs as arguments and it will more readable if they make the first parameter as place_t followed by comma separated DDFs